<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udpcilent</class>
 <widget class="QMainWindow" name="udpcilent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>udpcilent</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QListWidget" name="contentListWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>6000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverIPLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ray</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverIPLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>服务器地址</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>391</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>进入聊天室</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
